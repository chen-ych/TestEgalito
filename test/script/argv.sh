#!/bin/bash
mkdir -p tmp

ln -sf ../../src/libegalito.so
LD_LIBRARY_PATH=../../src EGALITO_DEBUG=/dev/null ../../src/loader \
	../binary/build/argv one two \
    >tmp/argv.out 2>&1
rm libegalito.so

if [ -z "$(diff argv.expected tmp/argv.out)" ]; then
    echo "test passed"
else
    echo "test failed!"
    exit 1
fi
