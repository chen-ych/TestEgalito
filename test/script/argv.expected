argv[0] = "../binary/build/argv"
argv[1] = "one"
argv[2] = "two"
argv[3] = "(null)"
